pkg/reconciler/v1alpha1/autoscaling/kpa/scaler_test.go

```diff
@@ -183,6 +184,54 @@ func TestScaler(t *testing.T) {
 	}
 }
 
+func TestEnableScaleToZero(t *testing.T) {
+	defer ClearAll()
+	tests := []struct {
+		label         string
+		startReplicas int
+		scaleTo       int32
+		minScale      int32
+		maxScale      int32
+		wantReplicas  int32
+		wantScaling   bool
+	}{{
```

Comment:
scaleTo can also be -1 because multiscaler doesn't send scale-to-zero when EnableScaleToZero is false

Reply:
Added a test case.